components/globals/AdminNav.tsx

```diff
@@ -21,6 +21,9 @@ const AdminNav = (props: AdminNavProps): React.ReactElement => {
         <li className="gc-horizontal-item">
           <Link href="/admin/vault">Form Vault</Link>
         </li>
+        <li className="gc-horizontal-item">
+          <Link href="/admin/users">Users</Link>
```

Comment:
Noticing that we don't have any translations for the admin page.

Reply:
When the page was originally built as a concept, and given the scope was purely internal at the time, we didn't do the translations.  I can add them quickly in this PR for the menu items.

Reply:
Or just create a new card so that it's on the list of things to do. No need to delay this PR.

Reply:
Added translations